src/js/global.js

```diff
@@ -156,6 +159,22 @@ ubuntu.globalNav = function() {
           }(moreList)
         );
       }
+
+      let smallScreenToggle = globalNav.querySelector('.global-nav__title');
+      let navList = globalNav.querySelector('.global-nav__list');
+      if (smallScreenToggle && navList) {
+        smallScreenToggle.addEventListener(
+          'click',
+          function(smallScreenToggle) {
+            return function(clickEvent) {
+              smallScreenToggle.classList.toggle('is-revealed');
+              smallScreenToggle.setAttribute('aria-expanded', true);
+              navList.classList.toggle('is-revealed');
+              navList.setAttribute('aria-hidden', false);
```

Comment:
Once clicked again this attribute doesn't toggle back to true. I think there might need to be a little bit of logic needed just to make sure the navList changes from false to true.